### PR TITLE
[FEATURE] Snippet that reverses a string

### PR DESCRIPTION
**Pull request by kshivi99**
## Description
Snippet that reverses the string.

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. 
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by kshivi99** [comment]
I could not understand why the deployment is failing. the logs are showing that there is some title error, but i have adhered to the guidelines. Let me know, a bit new to these things.

File: snippets/reverse_string.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: reverse_string
+tags: string,beginner
+---
+
+Returns the string after reversing it.
```
Comment (Chalarangelo) [inline snippets/reverse_string.md]: Maybe rephrase to:
```suggestion
Returns the reverse of a string.
```

**Comment by kshivi99** [comment]
@Chalarangelo Thanks for the review. Added the suggestion.
